doc/example_usage.rst

```diff
@@ -0,0 +1,12 @@
+==============================
+Example usage
+==============================
```

Comment:
This here is too long - should be a doc warning in travis.

Reply:
4cadc78dd05210eb5132d97ea88de7ab93c7b870